From 714d1a6dbb1ecda2b21e47e3073cc512c1e47438 Mon Sep 17 00:00:00 2001
From: "Yan, WalterX" <walterx.yan@intel.com>
Date: Wed, 11 Apr 2018 11:22:05 +0800
Subject: [PATCH] Decrease the duration of activity transition.

Modify the duration of animation to reduce the
duration of top status of an application.

Change-Id: I53be352c4b27d94e7aa05d6a444a8c37946ea570
Tracked-On:
Signed-off-by: Yan, WalterX <walterx.yan@intel.com>
---
 car_product/overlay/frameworks/base/core/res/res/anim/fade_in.xml  | 2 +-
 car_product/overlay/frameworks/base/core/res/res/anim/fade_out.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/car_product/overlay/frameworks/base/core/res/res/anim/fade_in.xml b/car_product/overlay/frameworks/base/core/res/res/anim/fade_in.xml
index ef5bcc4..038459c 100644
--- a/car_product/overlay/frameworks/base/core/res/res/anim/fade_in.xml
+++ b/car_product/overlay/frameworks/base/core/res/res/anim/fade_in.xml
@@ -17,5 +17,5 @@ limitations under the License.
     android:interpolator="@android:interpolator/decelerate_quad">
 
     <alpha android:fromAlpha="0.0" android:toAlpha="1.0"
-        android:duration="@android:integer/config_longAnimTime"/>
+        android:duration="250"/>
 </set>
diff --git a/car_product/overlay/frameworks/base/core/res/res/anim/fade_out.xml b/car_product/overlay/frameworks/base/core/res/res/anim/fade_out.xml
index 48c180b..42d76f3 100644
--- a/car_product/overlay/frameworks/base/core/res/res/anim/fade_out.xml
+++ b/car_product/overlay/frameworks/base/core/res/res/anim/fade_out.xml
@@ -17,5 +17,5 @@ limitations under the License.
     android:interpolator="@android:interpolator/decelerate_quad">
 
     <alpha android:fromAlpha="1.0" android:toAlpha="0.0"
-        android:duration="@android:integer/config_longAnimTime"/>
+        android:duration="250"/>
 </set>
-- 
1.9.1

